x-pack/plugins/security_solution/public/management/pages/policy/view/policy_details.tsx

```diff
@@ -41,15 +78,7 @@ export const PolicyDetails = React.memo(() => {
   );
 
   const backToEndpointList = (
-    <HeaderLinkBack
-      backOptions={{
-        text: i18n.translate('xpack.securitySolution.endpoint.policy.details.backToListTitle', {
-          defaultMessage: 'Back to endpoint hosts',
-        }),
-        pageId: SecurityPageName.endpoints,
-        dataTestSubj: 'policyDetailsBackLink',
-      }}
-    />
+    <BackToExternalAppButton {...backLinkOptions} data-test-subj="policyDetailsBackLink" />
```

Comment:
`HeaderLinkBack` (prior component) does not really play well with our route paths that are generated out of the `routing.ts` module because it does support a `pageId` of the top-level security app.